src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -94,6 +99,37 @@ internal static IntPtr LoadNativeLibrary()
             private static void PlatformInitialize()
             {
                 LibraryResolver.EnsureRegistered();
+
+                // Check whether the version of libgdiplus is a recent version of libgdiplus.
+                var installedVersion = LibgdiplusVersion;
+
+                if (installedVersion == null && didLoadNativeLibrary)
+                {
+                    throw new PlatformNotSupportedException(SR.Format(SR.LibgdiplusOutdated, LibraryRequiredVersion));
+                }
+                else if (installedVersion < LibraryRequiredVersion)
+                {
+                    throw new PlatformNotSupportedException(SR.Format(SR.LibgdiplusOutdated2, LibraryRequiredVersion, installedVersion));
```

Comment:
Is this message going to look good when `installedVersion` is null?

Reply:
There's an `if (installedVersion == null)` about two lines earlier, using another error message which does not mention the installed version.